src/SArray.jl

```diff
@@ -1,72 +1,70 @@
 """
-    SArray{Size, T, L}(x::NTuple{L, T})
-    SArray{Size, T, L}(x1, x2, x3, ...)
+    SArray{S, T, L}(x::NTuple{L, T})
+    SArray{S, T, L}(x1, x2, x3, ...)
```

Comment:
Ah, righto.  Yes, I find the constructor/type dichotomy odd too.  I often wish the doc system would provide more help here, in terms of autodocumenting the type signature docs, in a way which is systematic but not intrusive.  Oh ho, look at this trick:

```julia
module A

"""
    $(B)
"""
type B{N,T,X,Y,Z}
end

end

help?> A.B
  A.B{N,T,X,Y,Z}

  B is a thing with some type parameters
```

Kinda close?